model/executer.go

```diff
@@ -11,15 +13,14 @@ func (g GitCommander) Reflog(opt string) ([]byte, error) {
 }
 
 // Execute git log opts
-func (g GitCommander) Commitlog(opt string) ([]byte, error) {
-	return exec.Command("git", "log", opt).Output()
+func (g GitCommander) Commitlog(opts ...string) ([]byte, error) {
+	return exec.Command("git", "log", opts[0], opts[1]).Output()
```

Comment:
It is clear that golang crashes when a part of one argument contains with space.
At this time, I merge into master, but I want to improve in the future.
Could you create the issue?

Reply:
🤔

Reply:
✋ https://github.com/kamontia/qs/issues/131